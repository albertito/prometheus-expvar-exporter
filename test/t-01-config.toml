listen_addr = ":30080"

[target]
url = "http://localhost:30081/t-01-config.vars"

m.with_unsup.expvar = "with/unsupported-characters"
m.with_unsup.help = "This is very helpful"
