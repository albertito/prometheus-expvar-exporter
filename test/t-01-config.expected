# HELP a_float expvar "a-float"
# TYPE a_float untyped
a_float 1.2
# HELP an_int expvar "an-int"
# TYPE an_int untyped
an_int 26
# HELP with_unsup This is very helpful
# TYPE with_unsup untyped
with_unsup 55
